### PR TITLE
update to reflect newly added vendors

### PR DESCRIPTION
**Pull request by tyrp**

### CODE REVIEW

File: Setup/lead-form-tracking.html.md [hunk 1]
```diff
@@ -23,9 +23,13 @@ Here's how it works:
 1. Go to the Lead Form Tracking section of Compass settings and enable "Track lead forms automatically".
 	![Lead form tracking toggle](/images/leadform_1-disabled.png)
 2. Compass will automatically identify lead forms from the following providers:
-	* Mailchimp
+	* MailChimp
 	* Demandware
 	* Hubspot
```
Comment (bpapillon) [inline Setup/lead-form-tracking.html.md]: this one is also stylized as "HubSpot" I think

File: Setup/lead-form-tracking.html.md [hunk 2]
```diff
@@ -23,9 +23,13 @@ Here's how it works:
 1. Go to the Lead Form Tracking section of Compass settings and enable "Track lead forms automatically".
 	![Lead form tracking toggle](/images/leadform_1-disabled.png)
 2. Compass will automatically identify lead forms from the following providers:
```
Comment (bpapillon) [inline Setup/lead-form-tracking.html.md]: Are these in any order?

File: Setup/lead-form-tracking.html.md (hunk 1 above)
Comment (tyrp) [reply Setup/lead-form-tracking.html.md]: ah, yes.

File: Setup/lead-form-tracking.html.md (hunk 2 above)
Comment (tyrp) [reply Setup/lead-form-tracking.html.md]: Not currently, though probably would make sense to be alphabetical for readability

Comment (bpapillon) [reply Setup/lead-form-tracking.html.md]: alphabetical seems good, thanks

**Pull request merged**